examples/errors.js

```diff
@@ -7,19 +7,16 @@ const { emitWarning } = require('process')
 // Emit an `uncaughtException` event
 const uncaughtException = function() {
   setTimeout(() => {
-    // eslint-disable-next-line max-nested-callbacks
```

Comment:
The reason there were 2 setTimeout() was because by default Node.js calls process.exit() on uncaught exception. One example (before.js) does not use log-process-errors, i.e. use that default behavior. Not delaying the uncaught exception with an extra setTimeout() caused the example to exit too early. However I cannot reproduce this anymore, so removing that extra setTimeout() is a good idea.